lib/SILOptimizer/SILCombiner/SILCombinerCastVisitors.cpp

```diff
@@ -472,6 +472,9 @@ visitUncheckedTrivialBitCastInst(UncheckedTrivialBitCastInst *UTBCI) {
 SILInstruction *
 SILCombiner::
 visitUncheckedBitwiseCastInst(UncheckedBitwiseCastInst *UBCI) {
+  if (UBCI->getFunction()->hasOwnership())
```

Comment:
Isn't this too aggressive of a change? Shouldn't we still be able to do some of the simplifications below in OSSA?

Reply:
I think this is an unrelated change. It looks like it's actually my fault.I missed it in d3a01131d758f41ec132b98cece6153c7a7e240f.

Reply:
@gottesmm It doesn't have a real impact today. So should be fine ?
But will become important when we are moving OSSA beyond SILCombine.